packages/lib-classifier/src/plugins/tasks/DrawingTask/components/components/draggable/draggable.stories.js

```diff
@@ -0,0 +1,56 @@
+import { withKnobs } from '@storybook/addon-knobs'
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import React, { Component, createRef } from 'react'
+import sinon from 'sinon'
+import draggable from './draggable'
+
+function TestComponent ({ x, y }) {
+  return <circle fill='blue'r={10} cx={x} cy={y} />
+}
+
+const DraggableCircle = draggable(TestComponent)
+
+class DraggableStory extends Component {
+  constructor () {
+    super()
+    this.svg = React.createRef()
+    this.dragMove = this.dragMove.bind(this)
+    this.state={
+      x: 50,
+      y: 100
+    }
+  }
+
+  dragMove (e, d) {
+    const { x, y } = this.state
+    x += d.x
+    y += d.y
+    this.setState({ x, y})
+  }
+
+  render () {
+    const { children } = this.props
+    const { x, y } = this.state
+    const { dragMove, svg } = this
+    return (
+      <svg viewbox='0 0 300 400' height={300} width={400} ref={this.svg}>
+        {React.cloneElement(children, { dragMove, svg, x, y })}
+      </svg>
+    )
+  }
+}
+storiesOf('draggable', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    viewport: {
+      defaultViewport: 'responsive'
+    }
+  })
+  .add('basic', function () {
```

Comment:
This story doesn't seem to work for me. I can't drag the blue circle.

Reply:
If you look in the console, you'll see there's an error because the `<svg>` tag in the story isn't an `SVGElement` in the DOM. I have no idea why the browser isn't rendering it properly. Namespaces maybe?

Reply:
Or maybe it's a broken ref. I'm seeing that the `svg` ref is null.

Reply:
I see `Uncaught TypeError: Cannot read property 'createSVGPoint' of null` thrown in the console now.

Reply:
Maybe remove the story for now and add it later once the API is finalized?

Reply:
>I see Uncaught TypeError: Cannot read property 'createSVGPoint' of null thrown in the console now.

Yes, the SVG isn't an `SVGElement` so doesn't implement the SVG interfaces, including `createSVGPoint`. Wrong element namespace (HTML instead of SVG) maybe?

Reply:
Oh ignore that. `svg` is null so the ref is being created by the story.